src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -240,7 +234,11 @@ public function configureOptions(OptionsResolver $resolver)
                 $this->idReaders[$hash] = new IdReader($options['em'], $classMetadata);
             }
 
-            return $this->idReaders[$hash];
+            if ($this->idReaders[$hash]->isSingleId()) {
+                return $this->idReaders[$hash];
+            }
+
+            return null;
```

Comment:
Ha, this is exactly what I was trying to explain in our discussion :) good

Reply:
Yes this was implemented differently and lost in a rebase :)